exercise1/main.go

```diff
@@ -23,21 +28,22 @@ func main() {
 	}
 	defer filePtr.Close()
 
-	qaExtractor := fromcsv.Extractor{}
-	questions, answers, err := qaExtractor.Extract(filePtr)
+	store := &csv.Store{}
+	QAs, err := store.Extract(filePtr)
```

Comment:
Why do you need QAs in the main package if you have the Store as dependency in the Quiz?